### PR TITLE
build: migrate bots to cron.yaml

### PR DESCRIPTION
**Pull request by chingor13**
A few of these crons were never managed in source control. The `cron.yaml` format lets us specify multiple crons per bot.

See #1544

### CODE REVIEW

**Comment by chingor13** [comment]
Wait for #1843

File: packages/slo-stat-bot/cron.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+cron:
+  - name: slo_stat_bot
+    schedule: */5 * * * *
+    params:
+      installation:
+        id: 11112564
```
Comment (tmatsuo) [inline packages/slo-stat-bot/cron.yaml:6]: FYI, I don't see the installation id in the current payload.

**Review by tmatsuo — COMMENTED** [review]
I also noticed there are probably unnecessary jobs:
- label-sync
- sync-repo-settings

Should we manually delete them?

**Comment by chingor13** [comment]
> I also noticed there are probably unnecessary jobs:
> 
> * label-sync
> * sync-repo-settings
> 
> Should we manually delete them?

Yeah, duplicates were created because of the dasherized vs underscored names. The `cron` method used naming conventions, but the `cron.yaml` defines each scheduler job explicitly. The naming convenstion was fixed by #1843.

File: packages/merge-on-green/cron.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+cron:
+  - name: merge-on-green
+    schedule: */2 * * * *
+    description: merge on green for googleapis
+    params:
+      installation:
+        id: 6648093
+      org: googleapis
+      cleanUp: false
+      find_hangingprs: false
```
[the remark this answers is not in the document]
Comment (chingor13) [reply packages/merge-on-green/cron.yaml:10]: Will fix. Also, we should make the camel-casing vs. underscoring consistent, but that's something to do separately.

File: packages/slo-stat-bot/cron.yaml (hunk 1 above)
Comment (chingor13) [reply packages/slo-stat-bot/cron.yaml:6]: It's on the underscored one, not the dasherized (duplicate) job.

File: packages/policy/cron.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+cron:
+  - name: policy
+    schedule: 0 2 * * *
+    params:
+      installation:
+        id: 15630733
```
[the remark this answers is not in the document]
Comment (chingor13) [reply packages/policy/cron.yaml:6]: Added

**Review by tmatsuo — APPROVED** [review]

**Pull request merged**